tendermint/tests/integration.rs

```diff
@@ -143,4 +144,31 @@ mod rpc {
         // For lack of better things to test
         assert_eq!(status.validator_info.voting_power.value(), 10);
     }
+
+    #[tokio::test]
+    #[ignore]
+    async fn event_subscription() {
+        let mut client = tendermint::rpc::event_listener::EventListener::connect(
+            "tcp://127.0.0.1:26657".parse().unwrap(),
+        )
+        .await
+        .unwrap();
+        client.subscribe(&"tm.event='Tx'".to_owned()).await.unwrap();
+        // client.subscribe(&"tm.event='NewBlock'".to_owned()).await.unwrap();
+
+        // Collect and throw away the response to subscribe
+        let _ = client.get_event().await.unwrap();
+
+        // Loop here is helpful when debugging parsing of JSON events
+        // loop{
+        let resp = client.get_event().await.unwrap();
+        dbg!(&resp);
+        // }
+        match resp {
+            Event::GenericStringEvent { data: _ } => panic!(
+                "Expected GenericJSONEvent or JsonRPCTransactionResult, but got GenericStringEvent"
+            ),
```

Comment:
I do not fully understand why this is supposed to panic here? What is the purpose of `GenericStringEvent` if the integration tests only test that we panic if we see it? (previously was `assert!(false)` here).

Reply:
note that this integration test fails:
```
failures:

---- rpc::event_subscription stdout ----
[tendermint/tests/integration.rs:165] &resp = GenericStringEvent {
    data: "",
}
thread 'rpc::event_subscription' panicked at 'Expected GenericJSONEvent or JsonRPCTransactionResult, but got GenericStringEvent', tendermint/tests/integration.rs:168:54
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
(and it also failed before I simplified the match arm and the panic instead of `assert!(false)`)